install.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# check prerequisites NVM and AWS credentials
+if [ -d "${HOME}/.nvm/.git" ];
+then
+  echo "nvm installed";
+else
+  echo "nvm not installed. Install it as instructed here: https://github.com/nvm-sh/nvm#install--update-script";
```

Comment:
We should also run `aws sts get-caller-identity` to check if the terminal has AWS access.

Reply:
Added AWS check.